src/pybind/mgr/orchestrator/module.py

```diff
@@ -750,76 +678,70 @@ def _get_service_specs(self, service_name=None):
         return HandleCommandResult(stdout=specs)
 
     @_cli_write_command(
-        'orch apply mgr',
-        'name=placement,type=CephString,req=false',
-        'Update the size or placement of managers')
-    def _apply_mgr(self, placement=None):
+        'orch apply',
```

Comment:
having a `orch apply -i spec.yaml` using `ServiceSpec.from_json(yaml.load(buffer))` would be great.